src/notes/StructuredFieldPlugin.jsx

```diff
@@ -530,6 +560,7 @@ function StructuredFieldPlugin(opts) {
         utils: {
             //isSelectionInStructuredField
             convertSlateNodesToText: convertSlateNodesToText,
+            createStructuredField: createStructuredField.bind(null, opts)
```

Comment:
Not sure we need this line at all

Reply:
Not sure if the change for this got pushed, or do you still think we need this line?

Reply:
I forgot this push this. pushed now